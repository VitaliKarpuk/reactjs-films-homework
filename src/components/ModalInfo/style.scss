.modal-info {
  width: 449px;
  height: 235px;
  background-color: #000000a1;
  padding: 10px;
  position: relative;
  bottom: 42px;
  right: -105px;
  z-index: 7;

  p {
    height: 210px;
    font-size: 26px;
    line-height: 43px;
    color: #ffffff;
  }

  &::after {
    content: "";
    display: block;
    width: 50px;
    height: 50px;
    background-color: #000000a1;
    position: absolute;
    bottom: -25px;
    left: 200px;
    z-index: -1;
    transform: rotate(45deg);
    -webkit-transform: rotate(45deg);
    -moz-transform: rotate(45deg);
    -ms-transform: rotate(45deg);
    -o-transform: rotate(45deg);
  }
}
