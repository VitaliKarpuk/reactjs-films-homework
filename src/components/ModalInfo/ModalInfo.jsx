import React from 'react';

import './style.scss';

const ModalInfo = () => {
  return (
    <div className='modal-info'>
      <p>There are growing dangers in the wizarding world of 1926 New York.  Something mysterious is leaving a path of destruction in the streets, threatening to expose the wizarding</p>
    </div>
  )
}

export default ModalInfo;