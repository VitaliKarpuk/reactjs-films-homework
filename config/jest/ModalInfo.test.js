/* eslint-disable no-undef */
import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow';
import ModalInfo from '../../src/components/ModalInfo/ModalInfo';

describe('Test ModalInfo component', () => {
  test('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<ModalInfo className='modal-info' />);
    renderer.getRenderOutput();
    expect(result.type).toBe('div');
    expect(result.props.className).toBe('modal-info')
    expect(result.props.children.type).toBe('p');

    expect(result).toMatchSnapshot();
  })
})