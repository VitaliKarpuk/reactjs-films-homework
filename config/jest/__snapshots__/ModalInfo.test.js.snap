// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Adder adds two numbers 1`] = `
<div
  className="modal-info"
>
  <p>
    There are growing dangers in the wizarding world of 1926 New York.  Something mysterious is leaving a path of destruction in the streets, threatening to expose the wizarding
  </p>
</div>
`;
